src/app/_services/flag.service.ts

```diff
@@ -91,7 +91,8 @@ export interface FeatureFlags {
   solrMetadata?: boolean,
   // detailViews?: boolean,
   exportGoogle?: boolean,
-  relatedResFlag?: boolean
+  relatedResFlag?: boolean,
+  requireLoginForAdlDownload?: boolean
```

Comment:
So has this new flag already been initialized to `false` some how?

Reply:
Yes, it's set to false in the test-deployed aiw-ui.json file here: https://github.com/ithaka/artstor-flags/pull/145/files